cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -383,7 +383,8 @@ private static <C extends Connectable<C>> String getTapChangerId(C twt, String c
     }
 
     static boolean regulatingControlIsDefined(RatioTapChanger rtc) {
-        return !Double.isNaN(rtc.getTargetV())
```

Comment:
Here, I prefer that you keep previous code because reactive power control is not yet supported in CGMES conversion.